pkg/machine/applehv/machine.go

```diff
@@ -1145,6 +1145,7 @@ func generateSystemDFilesForVirtiofsMounts(mounts []machine.VirtIoFs) []ignition
 		mountUnit.Add("Mount", "What", "%s")
 		mountUnit.Add("Mount", "Where", "%s")
 		mountUnit.Add("Mount", "Type", "virtiofs")
+		mountUnit.Add("Mount", "Option", "defcontext=\"system_u:object_r:container_file_t:s0\"")
```

Comment:
Did you see https://github.com/containers/podman/issues/21269#issuecomment-1898539534 ?

I like using `nfs_t` here because I feel it better represents what's going on.

And actually if we go this route...would it potentially make sense instead of hardcoding security contexts to have something like `mount -o context-like=nfs` where the kernel looks up the default label for a `virtiofs` mount *as if* it was actually a `nfs` filesystem.

But anyways, none of this is blocking...I'm good with the patch as is.  (If you agree with my statements maybe we could add them as a `TODO` here at least)

Reply:
context-like would be getting patches into mount command which would be a lot more complex and would need to work in other places.

Theoretically the file system should be mounted as virtiofs_t, and have asked the upstream selinux team to figure out why it is showing up as unlabeled_t.

nfs_t is allowed for all containers as well.

Reply:
BTW defcontext will allow users to set the labels on the file system.

Reply:
Lima set the default to be nfs_t, so I guess we can go along with that.